### PR TITLE
fix: zookeeper backup size incorrect 

### LINKED ISSUE #1272: [BUG] zookeeper Backup size is  incorrect
The script to backup zookeeper is: 
```bash
java -cp /zoocreeper.jar  com.boundary.zoocreeper.Backup \
-z ${DP_DB_HOST}:${MY_ZOOKEEPER_ZOOKEEPER_SERVICE_PORT_CLIENT} --compress | \
datasafed push -z zstd-fastest - "${DP_BACKUP_NAME}.backup.json"

TOTAL_SIZE=$(datasafed stat / | grep TotalSize | awk '{print $2}')
echo "{\"totalSize\":\"$TOTAL_SIZE\"}" >"${DP_BACKUP_INFO_FILE}"
```

To fix:
1. update port MY_ZOOKEEPER_ZOOKEEPER_SERVICE_PORT_CLIENT --> DP_DB_PORT
2. the est size is incorrect, please check
```
TOTAL_SIZE=$(datasafed stat / | grep TotalSize | awk '{print $2}') # it report all files under path '/'
``` 

The size for backup is 20609627918, while the file size uploaded is just 4kb
1. backp info
image

2. check back up json

image

### PR DESCRIPTION
**Pull request by kubeJocker**
fix #1272

### CODE REVIEW

File: addons/zookeeper/templates/backupactionset.yaml [hunk 1]
```diff
@@ -6,6 +6,9 @@ metadata:
     {{- include "zookeeper.labels" . | nindent 4 }}
 spec:
   backupType: Full
+  env:
+    - name: DP_DB_PORT
```
Comment (shanshanying) [inline addons/zookeeper/templates/backupactionset.yaml:10]: DP_DB_PORT is injected by default

Comment (kubeJocker) [reply addons/zookeeper/templates/backupactionset.yaml:10]: Based on my check, this var DP_DB_PORT does not get automatically injected.

Comment (kubeJocker) [reply addons/zookeeper/templates/backupactionset.yaml]: Redeclare the variable as KB_CLIENT.

**Review by shanshanying — APPROVED** [review]

**Review by zjx20 — APPROVED** [review]

**Pull request merged**